src/cr/cube/cubepart.py

```diff
@@ -764,6 +771,10 @@ def table_margin(self):
     def table_margin_unpruned(self):
         return self._matrix.table_margin_unpruned
 
+    @lazyproperty
+    def table_moe(self):
+        return self.z_alpha * 100 * self.table_std_err
+
```

Comment:
The docstring here could be the same of the `columns_moe` but with table percentages reference:
```
"""Returns the margin of error (MoE) for table percentages"""
```

Reply:
Fixed all, and resolved conversations